src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $background-color-white;
+  margin-right: 48px;
+
+  &__img {
+    display: flex;
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+    top: 33px;
+  }
+
+  &__desciption {
+    display: flex;
+    justify-content: center;
+    margin: 0 16px;
+    font-weight: 500;
+    color: $text-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    margin-left: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__stars {
+    display: flex;
+    height: 16px;
+    margin-top: 16px;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__review {
+    display: flex;
+    text-align: right;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-right: 17px;
+    color: $text-color;
+  }
+
+  &__button {
+    display: flex;
```

Comment:
Let's organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)